src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSkinPresence.java

```diff
@@ -11,20 +11,20 @@
 
 public class TestAdsSkinPresence extends NewTestTemplate {
 
-  @Test(
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "skinWithoutTheme",
-      groups = "AdsSkinPresence"
-  )
-  public void adsSkinPresenceOnWikiWithoutTheme(String wikiName, String article,
-                                                Dimension resolution,
-                                                String expectedLeftSide,
-                                                String expectedRightSide,
-                                                String backgroundColor,
-                                                String middleColor) {
-    verifySkin(wikiName, article, resolution, expectedLeftSide, expectedRightSide, backgroundColor,
-               middleColor);
-  }
+//  @Test(
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "skinWithoutTheme",
+//      groups = "AdsSkinPresence"
+//  )
+//  public void adsSkinPresenceOnWikiWithoutTheme(String wikiName, String article,
+//                                                Dimension resolution,
+//                                                String expectedLeftSide,
+//                                                String expectedRightSide,
+//                                                String backgroundColor,
+//                                                String middleColor) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)